examples/tv-casting-app/tv-casting-common/core/CastingApp.cpp

```diff
@@ -114,6 +118,10 @@ CHIP_ERROR CastingApp::Start()
             });
     }
 
+    // Set a handler for Commissioner's CommissionerDeclaration messages.
+    chip::Server::GetInstance().GetUserDirectedCommissioningClient()->SetCommissionerDeclarationHandler(
+        CommissionerDeclarationHandler::GetInstance());
+
```

Comment:
Move this to CastingApp::PostStartRegistrations which specifically deals with doing all these kinds of registrations. Put this after the ChipDeviceEventHandler's registration.

Reply:
Okay